homeassistant/components/tessie/strings.json

```
@@ -529,5 +529,35 @@
     "command_failed": {
       "message": "Command failed, {message}"
     }
+  },
+  "services": {
+    "navigation_request": {
+      "name": "Set navigation destination",
```

Both actions set a destination, this one to an address, the other to some GPS coordinates.
So for proper translations I suggest to use "address" here instead of "destination".

```suggestion
      "name": "Set navigation address",
```